CHANGELOG.md

```diff
@@ -4,6 +4,56 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
 
 ---
 
+## 2.14.4
+
+
+### Bug Fixes
+- Code Security
+  - Ensures IAST propagation does not raise side effects related to `re.finditer`.
+- LLM Observability
+  - botocore: Fixes bedrock model and model provider interpretation from `modelId` when using cross-region inference.
+- Profiling
+  - Fixes an issue where stack v2 couldn't be enabled as pthread was not properly linked on some debian based images for aarch64 architecture.
+- Tracing
+  - Resolves the issue where tracer flares would not be generated if unexpected types were received in the `AGENT_CONFIG` remote configuration product.
+
+---
+
+## 2.13.2
+
+
+### Bug Fixes
+- Code Security
+  - Ensures IAST propagation does not raise side effects related to `re.finditer`.
+- LLM Observability
+  - botocore: Fixes bedrock model and model provider interpretation from `modelId` when using cross-region inference.
+- Profiling
+  - Fixes an issue where stack v2 couldn't be enabled as pthread was not properly linked on some debian based images for aarch64 architecture.
+- Tracing
+  - Resolves the issue where tracer flares would not be generated if unexpected types were received in the `AGENT_CONFIG` remote configuration product.
+
+---
+
+## 2.12.4
+
+
+### Bug Fixes
+- Profiling
+  - Fixes an issue where stack v2 couldn't be enabled as pthread was not properly linked on some debian based images for aarch64 architecture.
+- Tracing
+  - Resolves the issue where tracer flares would not be generated if unexpected types were received in the `AGENT_CONFIG` remote configuration product.
+
+---
+
+## 2.11.8
+
+
+### Bug Fixes
+
+- tracing(flare): Resolves the issue where tracer flares would not be generated if unexpected types were received in the AGENT_CONFIG remote configuration product.
```

Comment:
This one doesn't have the same format as the others above :eyes:

Reply:
Ah, good catch, thanks!